index.html

```diff
@@ -1,212 +1,319 @@
+        </section>
+      </section>
+    </main>
 
-            <section class="buts">
-                <details class="detbut" open>
-                    <summary>
-                        <span>Languages</span>
-                        <img src="/image2/Icon - Chevron - Down - Larger.svg" alt="" width="12" />
-                    </summary>
-                    <div class="divbut">
-                        <img src="/image2/Ellipse 1.png" alt="" />
-                        <span> javascript</span>
-                    </div>
-                    <div class="divbut">
-                        <img src="/image2/Ellipse 2.png" alt="" />
-                        <span> HTML</span>
-                    </div>
-                    <div class="divbut">
-                        <img src="/image2/Ellipse 3.png" alt="" />
-                        <span> CSS</span>
-                    </div>
-                </details>
-                <details class="detbut gg">
-                    <summary>
-                        <span>Frameworks</span>
-                        <img src="/image2/Union.png" alt="" height="8" width="4px"/>
-                    </summary>
-                </details>
-                <details class="detbut gg">
-                    <summary>
-                        <span>Skills</span>
-                        <img src="/image2/Union.png" alt=""  height="8" width="4px"/>
-                    </summary>
-                </details>
-            </section>
-            </section>
-        
-    
-</body>
-</html>
\ No newline at end of file
+    <!-- the form section -->
+    <section id="form-section">
+      <section class="secc aS">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </section>
+      <section class="secc bs">
+        <form action="https://formspree.io/f/xbjeokqq" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                name="user-name"
+                id="user-id-name"
+                placeholder="your name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="user-email"
+                id="user-id-email"
+                placeholder="youremail@gmail.com"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="user-message"
+                id="user-is-message"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                required
+                placeholder="write your message here"
+              ></textarea>
+            </li>
+          </ul>
+          <button type="submit" class="bb">Get in touch</button>
```

Comment:
- [ ] Great work adding the contact form, I would recommend that we follow the original designs from the figma template, as mentioned in the project requirement we have to stick with the original design as much as possible, I see currently you have `color: #396df2;` from the figma template for the get in touch button the correct property is `color: #6070FF;` also make the changes on the hover effect accordingly